controls/radgridview/events/column-resize-event.md

```diff
@@ -46,19 +46,19 @@ You can subscribe to the events either declaratively or runtime. **Example 1** s
 
 ## AutoGeneratingColumn
 
-When RadGridView's **AutoGenerateColumns** property is set to **True** (the default value), RadGridView will create a separate column for each public property of the bound objects and the **AutoGeneratingColumn** event will be triggered for each of those columns.
+When you set RadGridView's **AutoGenerateColumn** property to **True** (the default value), RadGridView creates a column for each public property of the bound objects and an **AutoGeneratingColumn** event is  triggered for each of those columns.
 
-The **GridViewAutoGeneratingColumnEventArgs** class provides you with a few useful properties:
+You can set the following properties of the **GridViewAutoGeneratingColumnEventArgs** class:
 
-* **Cancel**: By setting this to **True**, you can cancel the creation of the current column.
+* **Cancel**: Setting this to **True** cancels the creation of the current column.
```

Comment:
I believe using "current column" here makes it clearer that only the creation of the column that is currently being generated will be canceled. The other columns will still be generated (provided their creation is not canceled).